jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/CatalogResource.java

```diff
@@ -457,13 +457,13 @@ private SubscriptionEvent getLastEventBeforeDate(final UUID subscriptionId, fina
         SubscriptionEvent lastEventBeforeRequestedDate = null;
         for (final SubscriptionEvent subscriptionEvent : subscription.getSubscriptionEvents()) {
             if (lastEventBeforeRequestedDate == null) {
-                if (subscriptionEvent.getEffectiveDate().compareTo(requestedDate) > 0) {
+                if (subscriptionEvent.getEffectiveDate().compareTo(requestedDateTime) > 0) { //TODO_1375 - Is this correct or should we compare LocalDate?
```

Comment:
I think we should keep it the way we have it (using `DateTime`): The code now manipulates `DateTime` - if this causes an issue, then we can revisit.